lit-config.js

```diff
@@ -12,6 +13,16 @@ module.exports = {
     "Promise": false
   },
   "rules": {
+    "arrow-spacing": 2,
+    "no-confusing-arrow": 2,
+    "no-duplicate-imports": 2,
+    "no-useless-constructor": 2,
+    "no-var": 2,
+    "prefer-arrow-callback": 2,
+    "prefer-const": 2,
+    "prefer-spread": 2,
+    "prefer-template": 2,
+    "sort-imports": [2, { "ignoreCase": true }],
```

Comment:
Added some more rules Dave B and I went through.  Any preference on the import sort order?  Right now it's `{ "memberSyntaxSortOrder": ["none", "all", "multiple", "single"] }`

Reply:
Which, as an example, looks like this:
```
import 'd2l-icons/d2l-icon.js';
import 'd2l-icons/tier1-icons.js';
import { html, LitElement } from 'lit-element/lit-element.js';
import { buttonStyles } from './button-styles.js';
import { D2LButtonMixin } from './button-mixin.js';
import { ifDefined } from 'lit-html/directives/if-defined.js';
```

Reply:
LGTM!

Reply:
I _think_ so.  We can tweak as needed.  Are those last 3 all "single".  Wondering how easy it's going to be for folks to figure out the order they need to use.

Reply:
Yup the last three count as single, as would one without the `{ }` (like `import ifDefined } from 'lit-html/directives/if-defined.js';`

Reply:
If we turn the `ignoreCase` back off, it would need to be:
```
import 'd2l-icons/d2l-icon.js';
import 'd2l-icons/tier1-icons.js';
import { html, LitElement } from 'lit-element/lit-element.js';
import { D2LButtonMixin } from './button-mixin.js';
import { buttonStyles } from './button-styles.js';
import { ifDefined } from 'lit-html/directives/if-defined.js';
```

Reply:
Torn on which I prefer.  At first I wanted it to ignore the uppercase, but my editor highlights them differently so it does look a bit more random ignoring the case

Reply:
Talked with Dave B an Marg offline, we're going with this for now!